pyproject.toml

```diff
@@ -1,3 +1,56 @@
 [build-system]
-requires = ["py-cpuinfo==8.0.0", "setuptools", "wheel>=0.34.0"]
+requires = ["py-cpuinfo==9.0.0", "setuptools>=62.4.0", "wheel>=0.34.0"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "hdf5plugin"
+authors = [
+    {name = "ESRF - Data Analysis Unit", email = "silx@esrf.fr"}
```

Comment:
The name of the unit did change

Reply:
I stopped putting Data Analysis Unit or Software Group and use just ESRF.

Reply:
That sounds good, done